src/Symfony/Component/Console/Terminal.php

```diff
@@ -13,8 +13,9 @@
 
 class Terminal
 {
-    private static $width;
-    private static $height;
+    private $width;
+    private $height;
```

Comment:
I think they were static as a way to cache the values.

Reply:
Agreed, but only in a non application context. Application sets env `COLUMNS`/`LINES` from the calculated values in `Terminal`, which then provides caching since `Terminal` always checks the env first. Only those who instantiate `Terminal` outside of an application and did not reuse the instance would lose the cache. If this is not acceptable then I would have to use reflection to get the test coverage - is that OK?

Reply:
Looking at the code base, I can see that we are instantiating a Terminal instance several times (the instance is not shared).